src/widgets/AppWindow.js

```diff
@@ -772,13 +754,10 @@ export default class AppWindow extends createjs.Container {
     alert.addChild(alertLabel);
     alert.visible = false;
 
+    label = new createjs.Text('', '14px Arial');
```

Comment:
Since the creation of the instance is moved outside the callback, please move the register call for this label and the associated addChild calls to right after this line instead of in the callback.  Otherwise I see errors in the console when submitting multiple times.